control-plane/pkg/reconciler/trigger/v2/triggerv2.go

```diff
@@ -155,32 +197,47 @@ func (r Reconciler) reconcileConsumerGroup(ctx context.Context, broker *eventing
 		},
 	}
 
+	expectedCg.Spec.Replicas = ptr.Int32(1) //Must be set for Scaled Object ref
```

Comment:
Same concern as the other PR, in this way we will override KEDA changes when it changes the replicas.